defenz_alerts/main.py

```diff
@@ -235,12 +235,13 @@ def generate_email_content(report_details, block_type, network, interval):
     return email_body
 
 
-def _generate_malware_phishing_email_content(report_details):
+def _generate_malware_web_email_content(report_details):
```

Comment:
Maybe `_generate_malware_phishing_or_web_filter_email_content`?

Reply:
isn't that too long? I've put details in function doc string.

Reply:
It is long, but much cleaner. The short version requires to read the body of the function to be sure what it does.